src/ol/interaction/dragpaninteraction.js

```diff
@@ -144,7 +142,6 @@ ol.interaction.DragPan.handleDownEvent_ = function(mapBrowserEvent) {
     if (!this.handlingDownUpSequence) {
       view.setHint(ol.ViewHint.INTERACTING, 1);
     }
-    map.render();
```

Comment:
I wonder if this call and the call at L124 were about interrupting an animation.  Maybe that is not it, but it does seem like this is about providing notification that we are interacting (though it seems like an odd way to handle it).